docs/change-controlled-docs.rst

```diff
@@ -0,0 +1,233 @@
+
+Submitting a document to the CCB
+================================
+
+Establishing a new baselined version of the document begins with submitting it to the CCB.
+The release manager is responsible for orchestrating this process on behalf of DM.
+
+1. Choose a commit on ``master`` where the document's content is ready for CCB review.
+2. Create a DocuShare tag at the commit.
+   DocuShare tags are formatted as ``docushare-vN``, where ``N`` is the next-available document version number in DocuShare.
+3. Upload the PDF built by CI for the ``docushare-vN`` tag to DocuShare.
+   By definition, this uploaded document should become version ``N`` in DocuShare.
+4. For LDM documents, create a DM RFC.
+   For project documents (LSE), create an LCR.
+5. Once the RFC or LCR number is known, create a branch from the ``docushare-vN`` tag named ``tickets/RFC-N`` or ``tickets/LCR-N``.
+   This release branch initially has no new commits on it.
+
+.. figure:: ccd-submit.svg
+   :name: fig-ccd-submit
+   :align: center
+   :target: ccd-submit.svg
+   :alt: Git commit tree showing a docushare-v1 tag added to the head of the master branch.
+
+   Example Git tree showing document submission to the CCB.
+   The release manager creates a ``docushare-v1`` tag from the head of the ``master`` branch, since ``1`` is the next-available version number for the document in DocuShare.
+   The release manager submits the PDF built from the tag build to the CCB (as an RFC or an LCR).
+   In this case, the submission is ``RFC-1`` so the release manager creates the ``tickets/RFC-1`` release branch from ``docushare-v1`` tag.
+
+.. _ccd-edit:
+
+Addressing CCB feedback
+=======================
+
+The CCB may request amendments before the document can be baselined.
+
+1. Create a ticket branch from the ``tickets/RFC-N`` or ``tickets/LCR-N`` release branch.
+2. Commit amendments to that ticket branch and peer review.
+3. Rebase and merge the ticket branch onto the ``tickets/RFC-N`` or ``tickets/LCR-N`` release branch.
+4. Multiple ticket branches can be created and merged to the release branch if the CCB's requests are being addressed by multiple people working on separate parts of the document.
+5. The release manager creates a new DocuShare tag (``docushare-vN``) from the head of the release branch, uploads the revised document to DocuShare, and notifies the CCB.
+
+.. figure:: ccd-amend.svg
+   :name: fig-amend
+   :align: center
+   :target: ccd-amend.svg
+   :alt: Git commit tree showing amendments and resubmission to a CCB.
+
+   Example Git tree showing how CCB feedback is addressed on a release branch.
+   Two individuals address CCB feedback by creating separate ticket branches (``tickets/DM-3`` and ``tickets/DM-4``) from the release branch (``RFC-1``) that are merged back to the ``RFC-1`` release branch
+   The release manager resubmits the amended document to the CCB by creating a docushare tag (``docushare-v2``) and uploading version 2 of the document to DocuShare.
```

Comment:
For DM CCB we *could* say that the review process can use the .lsst.io RFC ticket branch version. Uploading the first draft to docushare and the final accepted version to docushare seems enough to me to satisfy LPM-51. I don't think LPM-51 requires that intermediate versions under review go through docushare. LSST CCB may not be as comfortable with that approach.

If the release manager is mediating all this by definition I don't want to add extra work that's not needed. If the author of the LDM thinks they have dealt with the review comments then they should be able to say so on the RFC and point at lsst.io without involving the release manager.